tests/test_results_api.py

```diff
@@ -16,6 +16,9 @@ def test_read_results(datadir, num_regression):
 
     results_metadata = read_metadata(datadir / RESULTS_FOLDER_NAME)
 
+    # TODO: arthur: We need a better way to identify the ids from the .alfacase
```

Comment:
Positional mapping?

```yaml
outputs:
  trends:
  - curve_names:  # This will be "project.study_container.item00001.output_options.trend_out_definition_container.item00001".
    - elevation
    - flow pattern
    element_name: Conn 1
    position:
      value: 243.21963890841974
      unit: m
    location: main
  - curve_names:  # This will be "project.study_container.item00001.output_options.trend_out_definition_container.item00002".
    - elevation
    - flow pattern
    element_name: Conn 1
    position:
      value: 664.3204331334222
      unit: m
    location: main
```